zaza/charm_lifecycle/deploy.py

```diff
@@ -402,7 +402,8 @@ def parse_args(args):
                         help='Model to deploy to',
                         required=True)
     parser.add_argument('-b', '--bundle',
-                        help='Bundle name (excluding file ext)',
+                        help='Path to bundle file '
+                        '(e.g. ./tests/bundles/groovy-victoria.yaml)',
```

Comment:
The original comment indicated that there was no `.yaml`.  Is this no longer the case?

Reply:
Yes, this was wrong, here is the path that this string takes:

https://github.com/openstack-charmers/zaza/blob/master/zaza/charm_lifecycle/deploy.py#L428
https://github.com/openstack-charmers/zaza/blob/master/zaza/charm_lifecycle/deploy.py#L358
https://github.com/openstack-charmers/zaza/blob/master/zaza/charm_lifecycle/deploy.py#L309
https://github.com/openstack-charmers/zaza/blob/master/zaza/charm_lifecycle/deploy.py#L332

So this gets directly fed into `if os.path.exists(bundle):`, so this requires the extension.

Reply:
That looks like it is tests for a conflict when looking up templates.  Does it not work if supplied with `-b ./some/bundle/path/bundle` rather than `-b ./some/bundle/path/bundle.yaml`?

Reply:
Uhm, this is currently used in the distro-regression testing and looks like: `--bundle keystone_v2_smoke:xenial-ocata`

Reply:
this is a different script: https://github.com/openstack-charmers/zaza/blob/master/zaza/charm_lifecycle/func_test_runner.py

Reply:
this is the proof basically: https://github.com/openstack-charmers/zaza/blob/master/zaza/charm_lifecycle/func_test_runner.py#L122